src/app/cli/src/cli_entrypoint/mina_cli_entrypoint.ml

```diff
@@ -234,8 +234,8 @@ let setup_daemon logger =
   and work_selection_method_flag =
     flag "--work-selection" ~aliases:[ "work-selection" ]
       ~doc:
-        "seq|rand Choose work sequentially (seq) or randomly (rand) (default: \
-         rand)"
+        "seq|rand|off Choose work sequentially (seq), randomly (rand), or \
```

Comment:
`off` is an unfortunate name here, it sounds like it turns the snark work off. Is there another name we could choose?

Reply:
Would `offset` be fine ?

Reply:
I changed for `offset`.